lib/gscraper/accounts/user.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Gscraper.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+  import GscraperWeb.Gettext
+
+  alias Gscraper.Accounts.Password
+
+  schema "users" do
+    field :username, :string
+    field :encrypted_password, :string
+
+    field :password, :string, virtual: true
+    field :password_confirmation, :string, virtual: true
+
+    timestamps()
+  end
+
+  @fields ~w(username password password_confirmation)a
+
+  @doc false
+  def changeset(user, attrs) do
+    user
+    |> cast(attrs, @fields)
+    |> validate_required(@fields)
+    |> unique_constraint(:username)
+    |> validate_password
```

Comment:
Small comment about code styling
```suggestion
    |> validate_password()
```

Reply:
Fixed in 6032768 but I'm wondering why we need to add the parentheses when there are no parameters passed to this method call?

Reply:
It's a style guide to add parentheses when calling a function without params 🙌